src/components/InfoSection.js

```diff
@@ -19,4 +21,15 @@ const InfoSection = ({ headline, sectionId, children }) => {
   );
 };
 
+InfoSection.propTypes = {
+  headline: PropTypes.string.isRequired,
+  sectionId: PropTypes.string,
+  children: PropTypes.element,
```

Comment:
For children you'd usually use the prop type `node`. `element` is a rendered React component, `node` in comparison is anything that can be rendered including a string, a number, and so on.

I think this can also be required since an info section without a body doesn't really make sense